### ISSUE TITLE
clipmap_bounds incorrectly set to map bounds not clip bounds

### ISSUE THREAD

**Issue by lklepner** [nypl-spacetime/nypl-warper#45]
The bounds of the Preview window, which are determined by the clipmap_bounds_merc value, seem to be the bounds of the raw map, not the clipped map, causing zoomToMaxExtent to be offcenter and too wide (framing for the uncropped map).

**Comment by lklepner** [comment]
I suspect that fixing this problem will greatly enhance the ability to accurately search by location https://github.com/nypl-spacetime/nypl-warper/issues/36

For example if you search for rectified maps by location using "Mahopac, NY" and then review the first page of results you'll find that none of the maps listed actually contain Mahopac -- they are maps of the surrounding area which have extraneous space around the display area which overlap Mahopac itself. This inaccuracy maps the search by location function much less useful.

**Comment by mgiraldo** [comment]
yeah we have been making some fixes on this. originally the warper only used the bbox (i guess) because not all maps have a clip mask. we're slowly working on making fixes. i agree that using clip mask if present is the better way
